replication/event.go

```diff
@@ -38,6 +38,11 @@ func (e *BinlogEvent) Dump(w io.Writer) {
 	e.Event.Dump(w)
 }
 
+// ByteLen returns the total size of the raw event data in bytes.
+func (e *BinlogEvent) ByteLen() int {
+	return len(e.RawData)
```

Comment:
lgtm. Just to confirm that without this change you can still use `len(e.RawData)` to get the same result?

Reply:
ping @dt8269